content/YubiHSM2/Usage_Guides/YubiHSM_2_Windows_Deployment_Guide/Configure_the_Primary YubiHSM_2_Device.adoc

```diff
@@ -0,0 +1,175 @@
+
+
```

Comment:
I'd prefer that we don't start with empty lines, rather have the heading on first line.

Reply:
I will remove them throughout.